src/System.Net.NetworkInformation/src/System/Net/StartIPOptions.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    [Flags]
+    internal enum StartIPOptions { Both = 3, None = 0, StartIPv4 = 1, StartIPv6 = 2 }
```

Comment:
I probably just grabbed this one a while back in order to get myself compiling at first, but it's not even being used in any of the configurations now. I'll remove.